plugins/pubsub/README.md

```diff
@@ -0,0 +1,48 @@
+PubSub Plugin
```

Comment:
Can you call it "Google Cloud Pub/Sub" ? It's not immediately obvious this is related to "Google Cloud" services otherwise. Like the AWS SNS plugin ... https://github.com/alerta/alerta-contrib/tree/master/plugins/sns#aws-sns-pluginhttps://github.com/alerta/alerta-contrib/tree/master/plugins/sns#aws-sns-plugin